yarn.lock

```diff
@@ -8246,7 +8246,7 @@ __metadata:
     typeface-lora: 0.0.72
     typeface-roboto-mono: 0.0.75
     typeface-titillium-web: 0.0.72
-    volto-blocks-widget: 3.4.1
+    volto-blocks-widget: 3.4.6
```

Comment:
Questo update di dipendenze e' voluto o out of scope?

Reply:
Sembra scorrelato, c'era un'altra PR dedicata.